subiquitycore/controllers/network.py

```diff
@@ -134,7 +134,7 @@ def __init__(self, app):
         super().__init__(app)
         self.apply_config_task = SingleInstanceTask(self._apply_config)
         if self.opts.dry_run:
-            self.root = os.path.abspath(".subiquity")
+            self.root = self.opts.output_base
```

Comment:
are we fine with `.subiquity` (i.e. the default value for `--output-base`) no longer being an absolute path now?
I don't know the reason why we resolved it into an absolute path before ; just wondering.

Reply:
Safer to keep it.